assemblyline/datastore/stores/es_store.py

```diff
@@ -393,24 +393,32 @@ def _get_task_results(self, task):
         except KeyError:
             return res['task']['status']
 
-    def _safe_index_copy(self, copy_function, src, target, body=None, min_status='yellow'):
-        ret = copy_function(src, target, body=body, request_timeout=60)
-        if not ret['acknowledged']:
-            raise DataStoreException(f"Failed to create index {target} from {src}.")
+    def _get_current_alias(self, index):
+        if self.with_retries(self.datastore.client.indices.exists_alias, name=index):
+            return (list(self.with_retries(self.datastore.client.indices.get_alias, index).keys())[0])
+        return None
```

Comment:
No need to change this, but FYI, you can also get the a key of a dict with something like
```python
key = next(iter(data), "default-value")
```
You can use next, to get the first value from a list with default, similar to `get` on a dictionary.